lib/connection.ts

```diff
@@ -270,11 +270,13 @@ export class Connection {
    *
    * @param message the message to start the subscription
    * @param callback the callback to be called when a new item arrives
+   * @param [resubscribe=true] re-established a subscription after a reconnect
    * @returns promise that resolves to an unsubscribe function
    */
   async subscribeMessage<Result>(
     callback: (result: Result) => void,
-    subscribeMessage: MessageBase
+    subscribeMessage: MessageBase,
+    resubscribe: boolean = true
```

Comment:
Let's make this an options object, in case we want to add more options in the future. 
```suggestion
    options?: { resubscribe?: boolean }
```